front-end/src/components/Post/GovernanceSideBar/Referenda/ReferendumVoteInfo.tsx

```diff
@@ -91,30 +96,40 @@ const ReferendumVoteInfo = ({ className, referendumId, threshold }: Props) => {
 	}, [electorate, nayVotes, threshold, turnout]);
 
 	return (
-		<Card className={className}>
-			<h3>Overview</h3>
-			<VoteProgress
-				ayeVotes={ayeVotes}
-				className='vote-progress'
-				passingThreshold={passingThreshold}
-				nayVotes={nayVotes}
-			/>
-			<Grid columns={3} divided>
-				<Grid.Row>
-					<Grid.Column>
-						<h6>Turnout</h6>
-						<div>{formatBnBalance(turnout, { numberAfterComma: 2 })}</div>
-					</Grid.Column>
-					<Grid.Column width={5}>
-						<h6>Aye</h6>
-						<div>{formatBnBalance(ayeVotes, { numberAfterComma: 2 })}</div>
-					</Grid.Column>
-					<Grid.Column width={5}>
-						<h6>Nay</h6>
-						<div>{formatBnBalance(nayVotes, { numberAfterComma: 2 })}</div>
-					</Grid.Column>
-				</Grid.Row>
-			</Grid>
+		<Card className={loadingStatus.isLoading ? `LoaderWrapper ${className}` : className}>
+			{loadingStatus.isLoading
+				?
+				<Loader text={loadingStatus.message}/>
+				: apiReady
```

Comment:
same here, no need to check for the `apiReady`, it's ready for sure in this state.